project/Extra.scala

```diff
@@ -8,11 +8,26 @@ import bintray.BintrayKeys.{bintrayRepository, bintrayOrganization}
 
 object Extra {
 
-  val sbtPluginSettings = ScriptedPlugin.scriptedSettings ++ Seq(
+  val sbtPluginSettings = ScriptedPlugin.scriptedSettings.filterNot(
+      _.key.key.label == libraryDependencies.key.label) ++ Seq(
+      // https://github.com/sbt/sbt/issues/3325
```

Comment:
OK that makes sense.

I will let @densh decide whether removing the scripted settings from `sbt-crossproject` makes sense. I am not too familiar with the consequences here.